packages/core/src/telemetry/detached-flush.js

```diff
@@ -19,7 +19,7 @@ async function detachedFlush() {
     events.map(async (event) => {
       await fetch(telemetryUrl, {
         method: "POST",
-        body: JSON.stringify(event.payload),
+        body: JSON.stringify(event),
```

Comment:
We want to include the whole event object, including event name and meta.

Reply:
Makes sense, think this was an oversight on my part!